### PR TITLE
Add note for max_map_count on Docker for Mac

### PR DESCRIPTION
**Pull request by thepill**
Add note for using it running Docker for Mac.
max_map_count must be set within the xhyve virtual machine

### CODE REVIEW

**Comment by antoineco** [comment]
The Docker for Mac VM already uses a high value for `max_map_count`, wouldn't it be better to simply mention it?

**Comment by thepill** [comment]
It didn't run on my machine (newest Docker release) without setting the `max_map_count` value

File: README.md [hunk 1]
```diff
@@ -30,10 +30,19 @@ Based on the official images:
...
+### Linux
 ```bash
 $ sudo sysctl -w vm.max_map_count=262144
 ```
 
+### Mac (Docker for Mac)
+```bash
+$ screen ~/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/tty
+```
+Log in with root and no password
```
Comment (antoineco) [inline README.md]: Not necessary on newer versions.

File: README.md [hunk 2]
```diff
@@ -30,10 +30,19 @@ Based on the official images:
...
+```bash
+$ sysctl -w vm.max_map_count=262144
+```
```
Comment (antoineco) [inline README.md]: A note about the procedure to detach screen would be useful (Ctrl+A - D)

**Comment by antoineco** [comment]
Ok fair enough then, I reviewed your changes.

Regarding the default value, are you sure you're using the latest version? On a fresh installation:
```
/ # grep max_map_count /etc/sysctl.d/01-moby.conf
vm.max_map_count = 262144
```

**Comment by antoineco** [comment]
Another suggestion: maybe replace the whole section with a link pointing to https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-cli-run-prod-mode

**Comment by deviantony** [comment]
+1 for the link to the elastic guide, existing section can be updated to point at the elastic docs.

**Comment by thepill** [comment]
I upgraded yesterday to version 17.03.0-ce-mac1 (15583).
1+ for the link - definitely the better option

**Comment by antoineco** [comment]
Thanks @thepill! :+1:

**Pull request merged**